server/data/db.go

```diff
@@ -12,11 +12,11 @@ var db *pgxpool.Pool
 
 func Connect() {
 
-	name := getEnv("POSTGRES_DB", "db")
-	password := getEnv("POSTGRES_PASSWORD", "pass")
-	host := getEnv("POSTGRES_HOST", "host")
-	port := getEnv("POSTGRES_PORT", "5000")
-	user := getEnv("POSTGRES_USER", "user")
+	name := os.getEnv("POSTGRES_DB")
+	password := os.getEnv("POSTGRES_PASSWORD")
+	host := os.getEnv("POSTGRES_HOST")
+	port := os.getEnv("POSTGRES_PORT")
+	user1 := os.getEnv("POSTGRES_USER")
```

Comment:
This is called `user1`, but `user`, which I don't think exists anymore, is passed as an argument to `Sprintf`. (This variable used to be called `user`)

Is VSCode providing any diagnostics, or does there already exists a variable named `user` somewhere else in the package?

Reply:
It was a typo from me, just fixed it